docker-compose.yml

```diff
@@ -0,0 +1,24 @@
+version: "2.1"
+services:
+
+  tribunals-case-api:
+    build:
+      context: .
+    image: docker.artifactory.reform.hmcts.net/sscs/tribunals-case-api
```

Comment:
is artifatory.reform.hmcts.net available for public? I am not 100% sure.Can you please confirm @timja-kainos ?

Reply:
no, but its an allowed exception to put here.